wandb/sdk/wandb_run.py

```diff
@@ -1359,32 +1359,64 @@ def _datatypes_callback(self, fname: str) -> None:
         files: FilesDict = dict(files=[(GlobStr(glob.escape(fname)), "now")])
         self._backend.interface.publish_files(files)
 
-    def _serialize_custom_charts(self, data: dict[str, Any]) -> dict[str, Any]:
-        if not data:
-            return data
-
-        chart_keys = set()
+    def _get_all_charts(
```

Comment:
Maybe `_pop_all_charts`? And document that this removes the charts from the given dictionary and returns a dictionary mapping them to dot-separated keys.